tools/reviewer/web/src/app/dashboard/reviews-panel/diff/diff.component.ts

```diff
@@ -1,14 +1,26 @@
+// NOTICE: it's actually not a diff, it's a two files changes
```

Comment:
> it's actually not a diff, it's a two files changes
What do you mean? Isn't a diff the changes between 2 files?

Reply:
We have `Diff` in out proto. It includes all files, changes, comment etc. But the component doesn't describe whole `Diff`, it describes one file only (and its changes)

Reply:
We have review.component, which actually describe `Diff`

Reply:
So maybe rename to `DiffFile`?